src/main/java/seedu/address/ui/HelpWindow.java

```diff
@@ -5,8 +5,11 @@
 import javafx.fxml.FXML;
 import javafx.scene.control.Button;
 import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
 import javafx.scene.input.Clipboard;
 import javafx.scene.input.ClipboardContent;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextFlow;
```

Comment:
Good job on maintaining checkstyle and ordering of importing by both groups and lexo order